install_config/sssd_for_ldap_failover.adoc

```diff
@@ -0,0 +1,421 @@
+= Setting up SSSD for LDAP Failover
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+{product-title} provides an
+xref:../install_config/configuring_authentication.adoc#LDAPPasswordIdentityProvider[authentication
+provider] for use with Lightweight Directory Access Protocol (LDAP) setups, but
+it can only connect to a single LDAP server. This can be problematic if that
+LDAP server becomes unavailable. System Security Services Daemon (SSSD) can be
+used to solve the issue.
+
+Originally designed to manage local and remote authentication to the host
+operating system, SSSD can now be configured to provide identity,
+authentication, and authorization services to web services like {product-title}.
+SSSD provides advantages over the built-in LDAP provider, including the ability
+to connect to any number of failover LDAP servers, as well as the ability to
+cache authentication attempts in case it can no longer reach any of those
+servers.
+
+The setup for this configuration is advanced and requires a separate
+authentication server (also called an *authenticating proxy*) that
+{product-title} will communicate with. This topic describes how to do this setup
```

Comment:
Avoid future tense unless quantifying with a "when": 

s/that {product-title} will/for {product-title} to